app/scripts/modules/amazon/src/serverGroup/details/sections/multipleInstanceTypesSubSection.less

```diff
@@ -0,0 +1,50 @@
+@import '~bootstrap/less/mixins/clearfix.less';
```

Comment:
I couldn't get the build to succeed with importing `commonImports.less ` 🤷‍♀️ 
I am guessing modules other than core should be importing from core and not dependencies directly? Any suggestions to keep this clean?